api/src/middleware/authenticate.ts

```diff
@@ -11,17 +11,33 @@ import isJWT from '../utils/is-jwt';
  */
 const authenticate: RequestHandler = asyncHandler(async (req, res, next) => {
 	req.accountability = {
+		maintenance: false,
+		maintenance_role: null,
```

Comment:
This shouldn't be on the accountability object, as it's not related to the current user